app/src/main/java/org/projectbuendia/client/ui/chart/PebbleExtension.java

```diff
@@ -192,6 +192,8 @@ public Object apply(Object input, Map<String, Object> args) {
                 for (Object item : (Collection) input) {
                     values.add(item instanceof ObsValue ? (ObsValue) item : null);
                 }
+            } else if (input instanceof ObsValue) {
```

Comment:
How did this change help fix the problem?

I ask because it was the original intention for this filter to not handle an individual value; format_obsvalue is supposed to take an individual value and format_obsvalues is supposed to take a list or array of values.